File: abc/abc126/abc126_b.cpp
#include <iostream>

using namespace std;

int main()
{
    int S;
    cin >> S;

    int YY = S / 100;
    int MM = S % 100;

    bool YYMM = false;
    if (1 <= MM and MM <= 12) {
        YYMM = true;
    }

    bool MMYY = false;
    if (1 <= YY and YY <= 12) {
        MMYY = true;
    }

    if (YYMM and MMYY) {
        cout << "AMBIGUOUS" << endl;
    } else if (YYMM) {
        cout << "YYMM" << endl;
    } else if (MMYY) {
        cout << "MMYY" << endl;
    } else {
        cout << "NA" << endl;
    }
}

File: abc/abc126/abc126_b.py
S = input()

former = int(S[:2])
latter = int(S[2:])

is_former_mm = (1 <= former <= 12)
is_latter_mm = (1 <= latter <= 12)

ans = ""
if is_former_mm and is_latter_mm:
    ans = "AMBIGUOUS"
elif is_former_mm:
    ans = "MMYY"
elif is_latter_mm:
    ans = "YYMM"
else:
    ans = "NA"

print(ans)
